src/applications/personalization/dashboard/components/DashboardAlert.jsx

```diff
@@ -35,16 +35,14 @@ const DashboardAlert = ({
             {subheadline}
           </div>
         )}
-        <h2 className="vads-u-font-size--lg vads-u-font-family--serif vads-u-margin--0">
-          {headline}
-        </h2>
+        <h3 className="vads-u-margin--0">{headline}</h3>
       </header>
       <section className="status vads-u-display--flex">
         <div className="status-icon-container vads-u-flex--auto">
           <i />
         </div>
         <div className="vads-u-flex--1">
-          <div className="vads-u-line-height--3 vads-u-font-size--lg vads-u-font-family--sans vads-u-font-weight--bold">
+          <div className="vads-u-line-height--3 vads-u-font-size--h3 vads-u-font-family--sans vads-u-font-weight--bold">
```

Comment:
@1Copenut There's also an outstanding issue around this element, which I changed from an `h` element in the last PR (this is the "Enrolled" or "Update" status text). Should this actually be an `h4`?

Reply:
This could be an H4, but that's not a strongly held opinion. We have the H3 above it as the place setting, so I believe users would arrow to the next item to gather their status.